src/collective/cover/setuphandlers.py

```diff
@@ -1,8 +1,21 @@
 # -*- coding: utf-8 -*-
+from collective.cover.config import PROJECTNAME
+from plone.dexterity.interfaces import IDexterityFTI
 from Products.CMFPlone import interfaces as Plone
 from Products.CMFQuickInstallerTool import interfaces as QuickInstaller
+from zope.component import queryUtility
 from zope.interface import implements
 
+import logging
+import pkg_resources
+
+try:
+    pkg_resources.get_distribution('plone.app.stagingbehavior')
+except pkg_resources.DistributionNotFound:
+    IStagingSupport = None
+else:
+    from plone.app.stagingbehavior.interfaces import IStagingSupport
```

Comment:
about your phrase `don't make me think`, I look different.. `else` clause for me looks like an alien.. always when I see it in a `try` block I need to remember what it means.. and when I remember what it means I mentally tie the try block with the else block and re-read everything.. so I need to think a lot more.